docs/core/distribution-packaging.md

```diff
@@ -99,6 +99,8 @@ The **shared** folder contains frameworks. A shared framework provides a set of
 
 The folders marked with `(*)` are used by multiple packages. Some package formats (for example, `rpm`) require special handling of such folders. The package maintainer must take care of this.
 
+- (18) **{dotnet_root}** is a shared root for all .NET major and minor versions. As a consequence, you will see all diffrent runtime versions, for all .NET major and minor versions, under a single folder on user's machine - `{dotnet_root}/shared/Microsoft.NETCore.App`. The same is true for all other folders that contain version sub-folders, i.e. `{dotnet_root}/shared/sdk`. It is strongly advised that the name of `{dotnet_root}` folder be version agnostic, i.e. simply `dotnet`.
```

Comment:
I'm not finding the `(18)` reference in the above layout.  The numbers here are cross references to the above layout.  I think 18 needs to be added.

Reply:
I see, let me check how to update the table. I was thinking of adding this new section to be the first one, but feared the big diff due to renumbering. I'll see if {0} is also possible.